scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -119,6 +110,32 @@ object AcceptAllPathFilter extends PathFilter {
 }
 
 object FileSource {
+  val LOG = LoggerFactory.getLogger(this.getClass)
+
+  val VerboseFileSourceLoggingKey = "com.twitter.scalding.filesource.verbose.logging"
+
+  def verboseLogEnabled(conf: Configuration): Boolean = conf.getBoolean(VerboseFileSourceLoggingKey, false)
```

Comment:
I can make it private. Is it better to use scalding.Config even if it's private? It seems that what I'll have access to is a Configuration -- does it make sense to convert that to a scalding.Config (probably not?)